src/types.rs

```diff
@@ -1123,15 +1121,14 @@ pub(crate) fn can_be_overflowed_type(
 }
 
 /// Returns `None` if there is no `LifetimeDef` in the given generic parameters.
```

Comment:
I know it's kind of a nit given this isn't public facing documentation, but this isn't technically true anymore with the change

Reply:
very true, can change when i'm back home

Reply:
what's the point of me changing all these names if i don't update the docs with them 😂